README.md

```diff
@@ -28,7 +28,7 @@ Come on in, the water's lovely. More help? Ping `jfire` or `bhousel` on
 
 ## Prerequisites
 
-* [Node.js](http://nodejs.org/) version 4 or newer
+* [Node.js](http://nodejs.org/) version 4 or newer, ([npm](https://www.npmjs.com/package/npm) version 3 or newer)
```

Comment:
Wow, that's odd: When I tried to run [these tests](https://travis-ci.org/openstreetmap/iD/jobs/163742017) here with node v4.6.0 + npm v2.15.9 (installed via `n`), I got an error:

```
> node build.js && node development_server.js

Rebuilding
[Error: Could not resolve 'd3-transition' from …/iD/modules/behavior/breathe.js]
Error loading resource file:///…/iD/dist/iD.js (203). Details: Error opening …/iD/dist/iD.js: No such file or directory
ReferenceError: Can't find variable: iD
```

In fact, after `npm install`ing with npm v2, the d3-transition module is nested inside the d3 one: `iD/node_modules/d3/node_modules/d3-transition`, while it's a top level module in npm v3: `iD/node_modules/d3-transition`